app/views/layouts/application.html.erb

```diff
@@ -3,6 +3,7 @@
   <head>
     <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>  
     <%= javascript_importmap_tags %>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0" />
```

Comment:
Consider moving the Google Fonts stylesheet link to the head section of the HTML document for better performance. This will ensure that the font styles are loaded before rendering the page.